app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,50 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  # new
+  def new
+    @post = Post.new
+  end
+
+  # create
+  def create
+    # redirect_to root_path unless @current_user
+    @post = Post.create!(post_params)
+    redirect_to "/posts/#{@post.id}"
```

Comment:
When dealing with RESTful actions, using path helpers in place of hard-coded paths makes your code a lot more readable.